pdata/pprofile/stacks.go

```diff
@@ -0,0 +1,31 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package pprofile // import "go.opentelemetry.io/collector/pdata/pprofile"
+
+import (
+	"errors"
+	"math"
+)
+
+var errTooManyStackTableEntries = errors.New("too many entries in StackTable")
+
+// SetStack updates a StackTable, adding or providing a stack and returns its
```

Comment:
```suggestion
// SetStack updates a StackSlice, adding or providing a stack and returns its
```

Reply:
Unless the `StackSlice` type should be renamed to `StackTable`? Looking at the proto the stack table is a repeated stack field 🤔

Reply:
I don't think we should rename the `StackSlice` type. Every slice in pdata is named `Slice` (`ByteSlice`, `Int64Slice`, ...).
The field is named `StackTable` only within the dictionary, to match the proto name.